src/test/suite/completion.test.ts

```diff
@@ -178,19 +178,10 @@ describe("Should do completion", () => {
     ).never();
   });
   it("should skip completion request on Tab key (indention in)", async () => {
-    await openADocWith("");
-
-    await emulationUserInteraction();
-
-    await vscode.commands.executeCommand("tab");
-
-    await triggerInline();
-
-    await emulationUserInteraction();
-
-    verify(
-      stdinMock.write(new SimpleAutocompleteRequestMatcher(), "utf8")
-    ).never();
+    await runSkipIndentInTest("javascript");
+  });
+  it("should skip completion request on Tab key (indention in) in golang where indentation is \t", async () => {
```

Comment:
what's the difference between golang and other langs?

Reply:
it just will use '\t' by default, it was easier than re-config the IDE indentation in the test